### DISCUSSION TITLE
about gcode host

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#813] [F.A.Q] [answered]
Hello, Luc. I used the ESP700 command to test the GCODE host functionality. Esp3d responded with two OK's and stopped. Does GCODE host have any special requirements for Marlin firmware or GCODE.

**Comment by luc-github** [comment, marked as answer]
do you use latest git? https://github.com/luc-github/ESP3D/commit/c0d347e3be587d35f2618f4d6911052526320639

there was a bug fixed recently

**Reply by liqijian101** [reply]
thanks .
